doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1728,16 +1736,62 @@ String notations
      concrete string expressed as a decimal.  They may not return
      opaque constants.
 
-The following errors apply to both string and numeral notations:
+.. note::
+   Number or string notations for parameterized inductive types can be
+   added by declaring an :ref:`abbreviation <Abbreviations>` for the
+   inductive which instantiates all parameters. See example below.
+
+The :n:`via` option applies to both string and number notations:
+
+:n:`via @qualid__ind using ( {+, @qualid__constant @qualid__constructor } )`
+   When using this variant, :n:`@qualid__type` no
+   longer needs to be an inductive. It is mapped to the
```

Comment:
It?  You mean "the number"?

Reply:
`qualid__type`, I replaced ". It" by "and", hope it's less ambiguous.